ompi/mca/mtl/psm2/mtl_psm2.h

```diff
@@ -14,6 +14,8 @@
  * Copyright (c) 2015      Intel, Inc. All rights reserved
  * Copyright (c) 2015-2017 Los Alamos National Security, LLC. All rights
  *                         reserved.
+ * Copyright (c) 2020      Intel, Inc. All rights reserved.
```

Comment:
Did you mean to put this copyright change on a file you didn't otherwise edit?

Reply:
Yeah, there was a change, then I realized it wasn't needed.

Reply:
Done.